packages/stems/src/styles/colors.ts

```diff
@@ -38,3 +38,4 @@ export type ColorValue =
   | 'accentPurple'
   | 'accentBlue'
   | 'specialLightGreen'
+  | 'darkmodeStaticWhite'
```

Comment:
staticWhite was supposed to be the same in light/dark, should it be in here?

Reply:
staticWhite should also be in here eventually, this add was for compatibility with the new banner color add